samples/unity-of-bugs/Assets/Editor/Builder.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
```

Comment:
BOM char?

Reply:
Indeed it was, I guess some files have BOM char while others don't